hack/verify-godeps.sh

```diff
@@ -128,7 +128,12 @@ pushd "${KUBE_ROOT}" 2>&1 > /dev/null
     echo "If you're seeing this locally, run the below command to fix your Godeps.json:"
     echo "patch -p0 < godepdiff.patch"
     echo "(The above output can be saved as godepdiff.patch if you're not running this locally)"
+    echo "(The patch file should also be exported as a build artifact if run through CI)"
     KEEP_TMP=true
+    if [[ -x godepdiff.patch && -d "${WORKSPACE}/_artifacts" ]]; then
```

Comment:
`${ARTIFACTS_DIR}` seems to be exported in https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh, but not in https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/verify-dockerized.sh (which this specific job seems to use https://github.com/kubernetes/test-infra/blob/1a6f132432670e55a1be25bca50b73735f76420e/jobs/config.json#L353)